packages/generator-electrode/generators/app/templates/gulpfile.js

```diff
@@ -1 +1,2 @@
+process.env.ES6_LINT = true;
```

Comment:
The `lint-server` task is using  [.eslintrc-node](https://github.com/electrode-io/electrode/blob/master/packages/electrode-archetype-react-app-dev/config/eslint/.eslintrc-node) which in turn is extending [es6-node](https://github.com/electrode-io/electrode/blob/master/packages/electrode-archetype-react-app-dev/config/eslint/.eslintrc-node#L3)